File: University Projects/Java Projects/Design and Programming Environments/Proiect/Model/src/main/java/app/model/Organizer.java
package app.model;

public class Organizer extends Entity<Integer>{
    private final String username;
    private final String password;

    public Organizer(Integer id, String username, String password) {
        super(id);
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }
}

File: University Projects/C# Projects/Design and Programming Environments/Proiect/model/Organizer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace model
{
    public class Organizer : Entity<int>
    {
        public string Username { get; }
        public string Password { get; }

        public Organizer(int id, string username, string password) : base(id)
        {
            Username = username;
            Password = password;
        }
    }
}
